posts/exploring-iot-infrastructure-support-in-aws-part1.md

```diff
@@ -0,0 +1,188 @@
+---
+authors:
+- Ben Edridge
+tags:
+- AWS
+- IoT
+- Melbourne
+- Australia
+- Greengrass
+date: 2018-07-19T11:41:57.000Z
+title: "Exploring AWS IoT Core and Greengrass Offerings"
+image: https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2018/07/esp_device.jpeg
+---
+
+According to [IEEE](https://iot.ieee.org/newsletter/march-2017/three-major-challenges-facing-iot.html). The biggest challenges and issues facing IoT networks is security, privacy, connectivity, compatibility, standardisation and intelligent actions/analysis. 
```

Comment:
I'd expand the IoT acronym, and probably replace 'is security' with 'are security'